AllReadyApp/Web-App/AllReady/Views/Activity/Activity.cshtml

```diff
@@ -120,34 +122,35 @@
             <div class="col-sm-12">
                 @if (!User.IsSignedIn())
                 {
-                    @if (!Model.IsClosed)
-                    {
-                        <h2>
-                            <a asp-action="Login" asp-controller="Account" asp-route-ReturnUrl="/Activity/@Model.Id" class="btn btn-volunteer">Volunteer</a>
-                        </h2>
-                    }
-                    else {
-                        <h2>This activity is closed</h2>
-                    }
+                    if (!Model.IsClosed)
+                     {
+                         <h2>
+                             <a asp-action="Login" asp-controller="Account" asp-route-ReturnUrl="/Activity/@Model.Id" class="btn btn-primary  btn-volunteer">Volunteer</a>
+                         </h2>
+                     }
+                     else
+                     {
+                         <h2>This activity is closed</h2>
+                     }
                 }
                 else
                 {
-                    @if (!Model.IsClosed)
-                    {
-                        <h2>
-                            <a href="#" data-toggle="modal" data-target="#volunteerForm" class="btn btn-volunteer" data-bind="visible: !enrolled()" style="display: none">Volunteer</a>
-                        </h2>
-                    }
-                    else
-                    {
-                        <h2 data-bind="visible: !enrolled()"> This activity is closed</h2>
-                    }
+                    if (!Model.IsClosed)
+                     {
+                         <h2>
+                             <button class="btn btn-primary btn-volunteer" data-bind="visible: !enrolled(), click: signupForActivity" style="display: none">Volunteer<br /> for this Activity</button>

```

Comment:
The Volunteer button shouldn't look different depending on whether you are signed in. I think just saying "Volunteer" is sufficient. I would also make the button orange to be consistent with the rest of the action buttons on the site.

Signed in:
![image](https://cloud.githubusercontent.com/assets/775229/13198980/01b99ad6-d7dd-11e5-87a2-97238f903d76.png)


Not signed in:
![image](https://cloud.githubusercontent.com/assets/775229/13198971/daa452ba-d7dc-11e5-81f1-1e9fafcf6df8.png)